File: 4080-SmallestMissingMultipleOfK/4080-SmallestMissingMultipleOfK.py
# Last updated: 25/08/2026, 19:27:37
class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        i = 1
        while (True):
            if (k*i) not in nums:
                return k*i

            i+=1

File: 4080-SmallestMissingMultipleOfK/4080-SmallestMissingMultipleOfK.cpp
// Last updated: 25/08/2026, 20:59:07
class Solution {
public:
    int missingMultiple(vector<int>& nums, int k) {
        unordered_set<int> s(nums.begin(),nums.end());
        int i = 1;
        while(true){
            int target = k*i;
            if (s.count(target)==0){
                return target;
            }
            i++;
        }

    }
};